src/ORM/Table.php

```diff
@@ -956,10 +956,54 @@ public function association($name)
      * $users = $this->getAssociation('Articles.Comments.Users');
      * ```
      *
-     * @param string $name the alias used for the association.
+     * Note that this method requires the association to be present or otherwise
+     * throws an exception.
+     * If you are not sure, use hasAssociation() before calling this method.
+     *
+     * @param string $name The alias used for the association.
      * @return \Cake\ORM\Association|null Either the association or null.
```

Comment:
Return type needs to be updated.

Reply:
I changed the wrong class method. I will fix this.